packages/react-reconciler/src/__tests__/ReactSuspenseWithNoopRenderer-test.js

```diff
@@ -963,33 +963,36 @@ describe('ReactSuspenseWithNoopRenderer', () => {
 
   // @gate enableCache
   it('resolves successfully even if fallback render is pending', async () => {
-    ReactNoop.render(
+    const root = ReactNoop.createRoot();
```

Comment:
Is this new?

Reply:
The `createRoot` method? No it's been there